Tools/autotest/default_params/copter-heli.parm

```diff
@@ -8,13 +8,14 @@ COMPASS_OFS3_Z 1.0000
 COMPASS_OFS_X 2.8749
 COMPASS_OFS_Y -21.8262
 COMPASS_OFS_Z -21.8422
+EK2_ENABLE 1.0000
```

Comment:
master is using EKF3 so I'm not sure we should be enabling EKF2

Reply:
I'm definitely in favor of keeping it at EKF3.. unless this is intended to test both somehow?